storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,32 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options that are not possible with in-tree volume plug-ins, such as volume snapshots.
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned to be migrated to the CSI driver at that time.
+====
+
+The AWS Elastic Block Store (EBS) CSI Driver Operator deploys the CSI driver needed for this extensibility in {product-title}. The Operator also provides the StorageClass that you can use to create persistent volume claims (PVCs).
```

Comment:
Is the StorageClass one of several ways you can create PVCs, or the _only_ way? If it's the only way, s/you can use/is required. If one of several ways, disregard this suggestion :)

Reply:
You _need_ a storage class to create a PVC, and this gives you the option of using the default class installed by the Operator. But in this case, we don't want to say "the required StorageClass" because it can be changed by the user. So I'll use "a default StorageClass."